crates/polars-plan/src/logical_plan/options.rs

```diff
@@ -126,6 +126,7 @@ pub struct FileScanOptions {
     pub rechunk: bool,
     pub file_counter: FileCount,
     pub hive_partitioning: bool,
```

Comment:
Lets put `hive_partitioning` and `hive_schema` in one struct and add that as a single field `hive_options`.

Reply:
Added a `HiveOptions` struct 👍